Frontend/confirm.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <!-- SEO -->
+    <meta name="description"
+        content="queue-rona is queue system that helps contain the spread of COVID in university dining halls.">
+    <meta name="author" content="queue-rona">
+
+    <title>queue-rona</title>
+
+    <!-- Favicon -->
+    <link rel="icon" href="img/urx.png" type="image/png">
+
+    <!-- CSS -->
+    <link rel="stylesheet" href="css/style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+
+  <!-- Bootstrap core CSS -->
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
+    integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
```

Comment:
How was this generated?

Reply:
That was from the Bootstrap template that I used for the landing page. I just copied and pasted everything in the head over to that HTML file.

That can be removed.